### DISCUSSION TITLE
Can GHOST output goes into or within other target environment rather than PostgreSQL?

### DISCUSSION THREAD

**Discussion by nasgz** [cmu-sei/GHOSTS#176] [Q&A] [answered]
Hi,

I have a question regarding the output traffic data. Can GHOST output goes into or within other target environment rather than PostgreSQL?

**Comment by sei-dupdyke** [comment, marked as answer]
Hi - 

Are you wanting to use a different database behind the API? If so, what do you suggest?

You could always export the data out of the API's postgres database into some other format, such as csv. Or you could gather the activity history files from the client (which are just delimited text files) and aggregate them somehow.

Just trying to understand the use case, please let me know.

**Reply by nasgz** [reply]
Can the outputs be ported into a live system, rather than being put into a database or file? (I'm looking for potential use case)

**Reply by sei-dupdyke** [reply]
What do you mean by live system?

**Reply by nasgz** [reply]
in real time, we're thinking about injecting the output straight into simulator (for example), rather than having to go via a database

**Reply by sei-dupdyke** [reply]
You could pull the data out of the API, but you'd have to script that.

**Reply by nasgz** [reply]
Great to know! many thanks with your feedback
